Source/ACE.Server/Managers/EmoteManager.cs

```diff
@@ -42,672 +42,6 @@ public void Cancel()
         }
 
         // Thanks to the original pioneers in the emulator community for help decoding these structures!
-        public void Execute(Emote emote, WorldObject target)
```

Comment:
This certainly makes EmoteManager a lot cleaner / easier to deal with, but it also greatly decreases the visibility of the example code.

Suggested to move this example code to a new file, EmoteManagerPrototype.cs perhaps?